tests/unit/test_extensionPoints.py

```diff
@@ -204,15 +407,28 @@ def handler2():
 		self.action.notify()
 		self.assertEqual(called, [handler1, handler2])
 
-	def test_kwargs(self):

```

Comment:
Did you intend to remove this test (and the later test_kwargs tests)? I guess this is all tested by the `callWithSupportedKwargs` tests, but I see you do have member function tests here and I wonder whether it might still be useful to make sure this works for sanity sake, given that this is where callers will hit us.